adl/config.adl

```diff
@@ -23,7 +25,9 @@ struct ToolConfig {
     /// The storage location for release zip files
     BlobStoreConfig releases;
 
-    Vector<DeployContext> deployContexts = [];
+    /// Static Config contexts
+    StringKeyMap<StaticConfigTopicName, ConfigContextSource> configContexts = {};
```

Comment:
A couple of suggested renames: 

    - StaticConfigTopicName => StaticConfigName
    - ConfigContextSource => JsonSource